docs/_source/components-overview/pop_syn/single_star.rst

```diff
@@ -21,85 +23,167 @@ STARPROPERTIES
+  * - ``log_LZ``
+    - log10 total burning power excluding LH and LHe and photodisintegrations devided by L_sun.
+  * - ``log_Lnuc``
+    - log10 total nuclear reaction luminosity (LH + LHe LZ) in L_sun.
+  * - ``c12_c12``
+    - log10 total luminosity for c12_c12 reaction in L_sun.
+  * - ``surf_avg_omega_div_omega_crit``
+    - Average surface omega divided by critical omega.
+  * - ``total_moment_of_inertia``
+    - Total moment of inertia in g*cm^2.
+  * - ``log_total_angular_momentum``
+    - log10 total angular momentum of the star g*cm^2*s^-1
+  * - ``spin``
+    - Angular momentum of the star in g*cm^2*s^-1 or dimensionless BH spin.
+  * - ``profile``
+    - Stellar profile from MESA. [not current supported for the initial-final interpolator]
 
 Additional scalar properties are added during the evolution depending on which steps the star has undergone. These properties are not stored in the history.
 
-.. csv-table:: Additional output
-   :header: "Properties", "Descriptions"
-   :widths: 50, 150
-  * - `natal_kick_array`
-    - The natal kick array for the star if it has undergone a SN.
-      [velocity, theta, phi, ?]
-  * - `SN_type`
+.. list-table:: Additional output
+   :header-rows: 1
+   :widths: 50 150
+
+  * - Properties
+    - Descriptions
+  * - ``natal_kick_array``
+    - | The natal kick array for the star if it has undergone a SN. 
+      | contains:
+      
+      * velocity
+      * theta
+      * phi
+      * ?
+
+  * - ``SN_type``
     - The supernova type of the star.
-  * - `f_fb`
+  * - ``f_fb``
     - The fraction of fallback mass.
-  * - `spin_orbit_tilt_first_SN`
+  * - ``spin_orbit_tilt_first_SN``
     - The spin-orbit tilt after the first SN, if the star has undergone a SN.
-  * - `spin_orbit_tilt_second_SN`
+  * - ``spin_orbit_tilt_second_SN``
     - The spin-orbit tilt after the second SN, if a second SN has occurred.
-  * - `m_disk_radiated`
+  * - ``m_disk_radiated``
     - The mass of the disk radiated in the collapse of the star.
+  * = ``m_disk_accreted``
+    - The mass of the disk accreted in the collapse of the star.
 
 
-TODO: add missing properties
-
 State options
 ~~~~~~~~~~~~~
 
-Star states are defined on their relative position on the HR diagram as follows:
-
-.. csv-table:: State
-   :header: "State", "Description"
-   :widths: 10, 30
-
-   `PreMS`, "The star is on the pre-main-sequence phase."
-   `MS`, "The star is in the Main Sequence phase."
-   `PostMS`, "The star is in the post Main Sequence phase."
-   `HeMS`, "The star is at the beginning of the He burning phase."
-   `PostHeMS`, "The star is evolving through the beginning of the He burning phase."
-   `WD`, "The star is a White Dwarf."
-   `NS`, "The star is a Neutron Star."
-   `BH`, "The star is a Black Hole."
-
-TODO: update the states to POSYDON v2.0.0
+Star states are defined by their burning and surface properties.
+These states are combined to describe the stellar state.
```

Comment:
What about citing Fragos et al. 2023, ApJS, 264, 46, Fig. 17 (and Table 3?) here to offer a schematic of how the states are constructed? If there's space, what about including that Fig. here as well?

Reply:
The v1 version for the states is out of date with the v2 code.